CovertInfiltration/Src/CovertInfiltration/Classes/XComGameState_CovertActionExpirationManager.uc

```diff
@@ -0,0 +1,118 @@
+//---------------------------------------------------------------------------------------
+//  AUTHOR:  statusNone
+//  PURPOSE: Manager class to keep track of covert infiltration expirations
+//---------------------------------------------------------------------------------------
+//  WOTCStrategyOverhaul Team
+//---------------------------------------------------------------------------------------
+
+class XComGameState_CovertActionExpirationManager extends XComGameState_BaseObject;
+
+struct ExpiringAction
```

Comment:
Rename the struct to `ActionExpirationInfo` - that would make more sense and won't cause confusion when you have both this struct and action itself as local variables (eg. `Update` below)